docker/gitlab-content-ci/Dockerfile

```diff
@@ -34,4 +34,15 @@ RUN  apt-get update -y \
   && gcloud --version \
   && gsutil --version
 
-
+# Installing goenv
+RUN git clone https://github.com/syndbg/goenv.git ~/.goenv \
+  && echo 'export GOENV_ROOT="$HOME/.goenv"' >> ~/.bashrc \
+  && echo 'export PATH="$GOENV_ROOT/bin:$PATH"' >> ~/.bashrc \
+  && echo 'eval "$(goenv init -)"' >> ~/.bashrc \
+  && echo 'export PATH="$GOROOT/bin:$PATH"' >> ~/.bashrc \
+  && echo 'export PATH="$PATH:$GOPATH/bin"' >> ~/.bashrc \
```

Comment:
Did you verify the environment variables you exported work as expected? Sometimes exporting the .bashrc doesn't propagate to gitlab and its better to use `ENV` when needed

Reply:
Yeah - otherwise `goenv install 1.13.8` would fail.
Using the ENV variable is a good idea, as it'll ensure it persists, however since goenv updates need to run after the `goenv init` script, so it's recommended to put in the `.bashrc` file - the caveat for that is that it also requires to run it in the image itself.
The same goes for `nvm` env.